apps/settings/defaults.py

```diff
@@ -0,0 +1,149 @@
+"""
+Top level settings file for all apps.
+
+The settings here overrides any setting previously loaded
+from the `metrics_service.settings`.
+"""
+
+from dynaconf import Dynaconf, post_hook
+
+extra_applications = [
+    "django_prometheus",
+    "rest_framework.authtoken",
+    "corsheaders",
+    "django_extensions",
+]
+"""Extra applications added after PSF templating."""
+
+dab_applications = [
+    "ansible_base.rest_filters",
+    "ansible_base.rest_pagination",
+    "ansible_base.jwt_consumer",
+    "ansible_base.resource_registry",
+    "ansible_base.rbac",
+    "ansible_base.activitystream",
+    "ansible_base.feature_flags",
+    "ansible_base.api_documentation",
+]
+"""Default DAB applications layd out from PSF, add/remove according to the project needs,
+adjust `pyproject` dab extra dependencies acording to apps added/removed here.
+"""
+
+project_applications = [
+    "apps.core",
+    "apps.dynamic_settings",
+    "apps.tasks",
+    "apps.dashboard",
+]
+"""List of applications from the apps/ folder."""
+
+
+INSTALLED_APPS = [
+    "dynaconf_merge_unique",  # DO NOT REMOVE THIS
+    *dab_applications,
+    *project_applications,
+    *extra_applications,
+]
+"""Final state of the INSTALLED_APPS that will merge with the rest of the settings."""
+
+DEVELOPER_MODE_ENABLED = False
+"""Relax the permission system for developers"""
+
+DEBUG = False
+"""Enable debug mode"""
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.SessionAuthentication",
+        "rest_framework.authentication.TokenAuthentication",
+        "rest_framework.authentication.BasicAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAuthenticated",
+    ],
+    "UNAUTHENTICATED_USER": None,
+    "UNAUTHENTICATED_TOKEN": None,
+    "DEFAULT_FILTER_BACKENDS": [
+        "rest_framework.filters.SearchFilter",
+        "rest_framework.filters.OrderingFilter",
+    ],
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+        "apps.core.renderers.ServiceBrowsableAPIRenderer",
+    ],
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
+    "PAGE_SIZE": 25,
+    "DEFAULT_VERSIONING_CLASS": "rest_framework.versioning.NamespaceVersioning",
+    "DEFAULT_VERSION": "v1",
+    "ALLOWED_VERSIONS": ["v1"],
+}
+"""REST framework settings."""
+
+SPECTACULAR_SETTINGS__TITLE = "metrics_service API"
+"""Title of Swagger the API documentation."""
+SPECTACULAR_SETTINGS__DESCRIPTION = "API documentation for the metrics_service"
+"""Description of Swagger the API documentation."""
+SPECTACULAR_SETTINGS__VERSION = "v1"
+"""Version of Swagger the API documentation."""
+SPECTACULAR_SETTINGS__COMPONENT_SPLIT_REQUEST = True
+"""Split components into request and response for generating clients."""
+
+ALLOWED_HOSTS = [
+    "localhost",
+    "127.0.0.1",
+]
```

Comment:
```suggestion
ALLOWED_HOSTS = ['*']
```
needed for working behind gateway